### ISSUE TITLE
Don't get stuck on existing resolver-old

### ISSUE THREAD

**Issue by robcza** [whalebone/local-resolver-agent#8]
if there is a container `resolver-old` already, don't give up the upgrade and remove the old container:
```
2018-07-12 21:45:30,759 | 311 | INFO | 409 Client Error: Conflict ("Error when allocating new name: Conflict. The container name "/resolver-old" is already in use by container "f206df7c6ba28142dd39f82ef4c18d38c6cf2e68c175a0f9d555639a95a78d7c". You have to remove (or rename) that container to be able to reuse that name.")
```

Otherwise the agent will end up in a situation that it has to recovered manually.

**Comment by Narzhan** [comment]
Should be fixed in version 1.2.4

**Comment by robcza** [comment]
Tested, works.
Thank you
